packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -73,12 +96,34 @@ export const SelectablePill = (props: SelectablePillProps) => {
     }
   }
 
-  return (
-    <SelectablePillRoot {...props}>
+  const pillContent = (
+    <>
       {Icon ? <Icon css={iconCss} /> : null}
```

Comment:
just curious, would it be bad to put the icon inside the text?

Reply:
oh good q, probably not honestly, ill do it

Reply:
it was getting annoying having to make text a flex item to have the icon centered correctly so i ended up leaving it the same this time